Fails to build with FFmpeg 8

### Steps to reproduce

FreeBSD amd64

```
FAILED: Telegram/CMakeFiles/lib_ffmpeg.dir/SourceFiles/ffmpeg/ffmpeg_utility.cpp.o
/usr/bin/c++ -DCRL_FORCE_COMMON_QUEUE -DGI_INLINE -DGLIB_VERSION_MAX_ALLOWED=GLIB_VERSION_2_56 -DGLIB_VERSION_MIN_REQUIRED=GLIB_VERSION_2_56 -DKCOREADDONS_LIB -DQT_CORE_LIB -DQT_DBUS_LIB -DQT_DEPRECATED_WARNINGS_
SINCE=0x051500 -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_NO_CAST_FROM_BYTEARRAY -DQT_NO_DEBUG -DQT_NO_KEYWORDS -DQT_OPENGLWIDGETS_LIB -DQT_OPENGL_LIB -DQT_QMLINTEGRATION_LIB -DQT_QMLMETA_LIB -DQT_QMLMODELS_LIB -DQT_QMLW
ORKERSCRIPT_LIB -DQT_QML_LIB -DQT_QUICKWIDGETS_LIB -DQT_QUICK_LIB -DQT_SVG_LIB -DQT_WAYLANDCOMPOSITOR_LIB -DQT_WIDGETS_LIB -I/wrkdirs/usr/ports/net-im/telegram-desktop/work/tdesktop-6.0.2-full/Telegram/SourceFile
s -I/wrkdirs/usr/ports/net-im/telegram-desktop/work/tdesktop-6.0.2-full/Telegram/lib_base -I/wrkdirs/usr/ports/net-im/telegram-desktop/work/tdesktop-6.0.2-full/Telegram/lib_rpl -I/wrkdirs/usr/ports/net-im/telegra
m-desktop/work/tdesktop-6.0.2-full/Telegram/lib_crl -I/wrkdirs/usr/ports/net-im/telegram-desktop/work/tdesktop-6.0.2-full/cmake/external/glib/cppgir -I/wrkdirs/usr/ports/net-im/telegram-desktop/work/tdesktop-6.0.
2-full/cmake/external/glib/cppgir/gi -I/wrkdirs/usr/ports/net-im/telegram-desktop/work/tdesktop-6.0.2-full/cmake/external/glib/cppgir/override -I/wrkdirs/usr/ports/net-im/telegram-desktop/work/tdesktop-6.0.2-full
/cmake/external/glib/cppgir/expected-lite/include -I/wrkdirs/usr/ports/net-im/telegram-desktop/work/tdesktop-6.0.2-full/Telegram/lib_ui -I/wrkdirs/usr/ports/net-im/telegram-desktop/work/tdesktop-6.0.2-full/Telegr
am/lib_ui/emoji_suggestions -isystem /usr/local/include/qt6/QtCore -isystem /usr/local/include/qt6 -isystem /usr/local/lib/qt6/mkspecs/freebsd-clang -isystem /usr/local/include/qt6/QtCore/6.9.1 -isystem /usr/loca
l/include/qt6/QtCore/6.9.1/QtCore -isystem /usr/local/include/qt6/QtGui -isystem /usr/local/include/qt6/QtGui/6.9.1 -isystem /usr/local/include/qt6/QtGui/6.9.1/QtGui -isystem /usr/local/include/qt6/QtWidgets -isy
stem /usr/local/include/qt6/QtWidgets/6.9.1 -isystem /usr/local/include/qt6/QtWidgets/6.9.1/QtWidgets -isystem /usr/local/include/qt6/QtNetwork -isystem /usr/local/include/qt6/QtSvg -isystem /usr/local/include/qt
6/QtOpenGL -isystem /usr/local/include/qt6/QtOpenGLWidgets -isystem /usr/local/include/qt6/QtDBus -isystem /usr/local/include/qt6/QtQuick -isystem /usr/local/include/qt6/QtQml -isystem /usr/local/include/qt6/QtQm
lIntegration -isystem /usr/local/include/qt6/QtQmlMeta -isystem /usr/local/include/qt6/QtQmlModels -isystem /usr/local/include/qt6/QtQmlWorkerScript -isystem /usr/local/include/qt6/QtQuickWidgets -isystem /usr/lo
cal/include/qt6/QtWaylandCompositor -isystem /usr/local/share -isystem /usr/local/include/KF6/KCoreAddons -isystem /wrkdirs/usr/ports/net-im/telegram-desktop/work/.build/Telegram/lib_base/gen -isystem /wrkdirs/us
r/ports/net-im/telegram-desktop/work/.build/gen -isystem /usr/local/include/gio-unix-2.0 -isystem /usr/local/include/glib-2.0 -isystem /usr/local/lib/glib-2.0/include -isystem /wrkdirs/usr/ports/net-im/telegram-d
esktop/work/.build/Telegram/lib_ui/gen -O2 -pipe -march=znver4 -fstack-protector-strong -isystem /usr/local/include -fno-strict-aliasing  -isystem /usr/local/include -O2 -pipe -march=znver4 -fstack-protector-stro
ng -isystem /usr/local/include -fno-strict-aliasing  -isystem /usr/local/include  -DNDEBUG -std=gnu++20 -fPIC -fno-strict-aliasing -pipe -Wall -Wextra -Wno-unused-parameter -Wno-switch -Wno-missing-field-initiali
zers -Wno-sign-compare -Wno-deprecated -pthread -Wno-deprecated-declarations -MD -MT Telegram/CMakeFiles/lib_ffmpeg.dir/SourceFiles/ffmpeg/ffmpeg_utility.cpp.o -MF Telegram/CMakeFiles/lib_ffmpeg.dir/SourceFiles/f
fmpeg/ffmpeg_utility.cpp.o.d -o Telegram/CMakeFiles/lib_ffmpeg.dir/SourceFiles/ffmpeg/ffmpeg_utility.cpp.o -c /wrkdirs/usr/ports/net-im/telegram-desktop/work/tdesktop-6.0.2-full/Telegram/SourceFiles/ffmpeg/ffmpeg
_utility.cpp
/wrkdirs/usr/ports/net-im/telegram-desktop/work/tdesktop-6.0.2-full/Telegram/SourceFiles/ffmpeg/ffmpeg_utility.cpp:686:29: error: use of undeclared identifier 'av_stream_get_side_data'
  686 |         const auto displaymatrix = av_stream_get_side_data(
      |                                    ^
1 error generated.
```

### Expected behaviour

Builds OK

### Actual behaviour

Broken

### Operating system

FreeBSD

### Version of Telegram Desktop

6.0.2

### Installation source

Static binary from official website

### Crash ID

_No response_

### Logs

```shell

```

--- a/Telegram/SourceFiles/ffmpeg/ffmpeg_utility.cpp
+++ b/Telegram/SourceFiles/ffmpeg/ffmpeg_utility.cpp
@@ -683,10 +683,10 @@ int DurationByPacket(const Packet &packet, AVRational timeBase) {
 }
 
 int ReadRotationFromMetadata(not_null<AVStream*> stream) {
-	const auto displaymatrix = av_stream_get_side_data(
-		stream,
-		AV_PKT_DATA_DISPLAYMATRIX,
-		nullptr);
+	const auto displaymatrix = av_packet_side_data_get(
+		stream->codecpar->coded_side_data,
+		stream->codecpar->nb_coded_side_data,
+		AV_PKT_DATA_DISPLAYMATRIX);
 	auto theta = 0;
 	if (displaymatrix) {
 		theta = -round(av_display_rotation_get((int32_t*)displaymatrix));

